torch/csrc/distributed/rpc/script_rref_proto.h

```diff
@@ -0,0 +1,68 @@
+#pragma once
+
+#include <torch/csrc/distributed/rpc/message.h>
+#include <torch/csrc/jit/operator.h>
+#include <torch/csrc/jit/pickler.h>
+#include <vector>
+
+namespace torch {
+namespace distributed {
+namespace rpc {
+
+// Temporary solution of RRef operations.
+// TODO: Remove all these messages and use rpc + registered functions instead.
```

Comment:
What registered functions? You mean JIT built-ins?

Reply:
I think RRef should be built on top of RPC not in RPC. It does not has to be JIT function, we could have our own namespace for RRef. Similar to using `torch.ops.aten.*` for builtin ops, we could have `torch.distributed.rpc.*` for RPC related functions. Then all these messages can be unified as a `SCRIPT_CALL`.